amundsen_application/static/js/components/TableDetail/SourceLink/index.tsx

```diff
@@ -16,8 +16,22 @@ const SourceLink: React.FC<SourceLinkProps> = ({
 }: SourceLinkProps) => {
   if (tableSource === null || tableSource.source === null) return null;
 
-  const image =
-    tableSource.source_type === 'github' ? '/static/images/github.png' : '';
+  // const image =
+  //   tableSource.source_type === 'github' ? '/static/images/github.png' : '';
+
+  let image;
+  switch (tableSource.source_type) {
+    case 'github': {
+      image = '/static/images/github.png';
+      break;
+    }
+    case 'gql': {
```

Comment:
yes I can put this in private config. But I do not think this is specific to Lyft as Table lineage because we are not creating any url/ pattern specific to Lyft. It is more similar to adding new database which can be used by any company. Image repo provided here is path to images folder which is inside public repo.

Reply:
Do you think I should configure it similar to Mode and redash icon as done for dashboards? e.g https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config-default.ts#L57

Reply:
if we think graphql generates table could be generic, do you think we could have a small doc about which case we could leverage this new source link?

This could help to adopt this new source icon.

Reply:
I am totally ok to put this in the repo, but would like have a doc to document this feature and how users could use it.

Reply:
yeah I will add documentation for this feature. We can keep github as default table source in public configurations and new table sources can be added to private configurations.